torchvision/models/googlenet.py

```diff
@@ -193,8 +194,7 @@ def _forward(self, x):
         return x, aux2, aux1
 
     @torch.jit.unused
-    def eager_outputs(self, x, aux2, aux1):
-        # type: (Tensor, Optional[Tensor], Optional[Tensor]) -> GoogLeNetOutputs
+    def eager_outputs(self, x: Tensor, aux2: Tensor, aux1: Optional[Tensor]) -> Union[GoogLeNetOutputs, Tensor]:
```

Comment:
Not sure if I can use `Union` when the method is decorated with `@torch.jit.unused`. If not, we can only ignore this error.

Reply:
I'm not 100% sure. Tests seem to pass, so it should be fine?

cc @eellison to double-check

Reply:
Yea, the type does get used so I don't think you can use union. Can try `Any` maybe ?

Reply:
Thanks for confirming @eellison , although I'm a bit worried why torchscript tests passed then.

I would say just keep the old type annotation as before, and forget the `Tensor` return type?

Reply:
Actually i could be wrong... let me double check sorry

Reply:
Okay, it does look at annotations.

Reply:
I went for the ignore for now.